refactor(app): Update tip length cal videos

sfoster1 opened the pull request:
These videos don't show the rest of the deck slots in the background,
and therefore are safe to use no matter where the calibration block is.
This is required for health check because the calibration block is in a
different place.

sfoster1 committed:
refactor(app): Update tip length cal videos

These videos don't show the rest of the deck slots in the background,
and therefore are safe to use no matter where the calibration block is.
This is required for health check because the calibration block is in a
different place.

ahiuchingau reviewed (commented):
For cal check, these new videos work perfectly for both mounts. However, for tip length calibration on the right pipette, the orientation of the block is not correct (tall side should be on the right).

Do we want to keep the older version of the videos for tlc, and only use the new ones for cal check?

sfoster1 commented:
@ahiuchingau Yeah I think we should do that

sfoster1 committed:
fixup: separate assets btwn tlc and healthcheck

Since it turns out you can't actually use these for tlc because the
calblock is flipped the wrong way

ahiuchingau reviewed (approved):
LGTM!
